assets/js/event-providers/woocommerce.js

```diff
@@ -19,13 +19,171 @@
 		return;
 	}
 
+	const currency = global._googlesitekit.wcdata.currency;
```

Comment:
Let's use `?.` for consistency because in some places in this script we use it like in `global._googlesitekit?.gtagEvent?.( ... )`, in other don't.